### DISCUSSION TITLE
Where is configuration directory on Windows?

### DISCUSSION THREAD

**Discussion by Jaehaks** [mucommander/mucommander#1230] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues (including the closed ones)

### Description

After I install mucommander, configuration directory does not exist. 
Wiki said that the directory is ~/.mucommander in windows standalone installer. 
But the directory isn't generated automatically. 

When I set some configurations, these are saved but I don't know what files save the data. 
There aren't any .xml file for setting. but mucommander is remember

### Steps to reproduce

After install mucommander

### Expected behavior

.mucommander directory is generated in home folder

### Actual behavior

configuration directory does not exist

### muCommander version

1.4.1, latest

### Java version

1.8.0_411

### Operating System type and version

Windows 10

**Comment by ahadas** [comment]
@Jaehaks this was originally filed as a bug but from the description it seems that everything goes well for you, however, you can't find the directory with the configuration files and therefore I converted the issue to a Q&A discussion.
the directory with the configuration files is named `.mucommander` and it resides within the user's home folder, as you quoted from the wiki. for example, for user `USER` its path should be: `C:\Users\USER\.mucommander`
